assets/css/amp-stories.css

```diff
@@ -89,14 +89,6 @@ table.wp-block-table {
 	background-image: url("../images/quote-white.svg");
 }
 
-/**
- * See https://github.com/ampproject/amp-wp/issues/2283
- */
-.edit-post-layout[data-block-name="core/video"] .edit-post-settings-sidebar__panel-block .block-editor-block-inspector__card + div > .components-panel__body:first-child .components-toggle-control:nth-child(2),
-.edit-post-layout[data-block-name="core/video"] .edit-post-settings-sidebar__panel-block .block-editor-block-inspector__card + div > .components-panel__body:first-child .components-toggle-control:nth-child(6) {
-	display: none;
-}
-
```

Comment:
This hid the 'Autoplay' and 'Play inline' toggles, but the edit component doesn't have them anymore.